lib/jelix/forms/jFormsDatasource.class.php

```diff
@@ -177,6 +177,9 @@ public function setCriteriaControls($criteriaFrom = null){
  */
 class jFormsDaoDatasource extends jFormsDynamicDatasource {
 
+    if ($key === null || $key == "")
+        return null;
+    
     protected $selector;
```

Comment:
Sorry, but you made the change oustide a method, in the class block...

Reply:
Oups ! I will make the correction as soon as possible...
I'm very sorry